### PR TITLE
auto for ocp-19922

### PR DESCRIPTION
**Pull request by zhouying7780**
Check point: Terminating pods should not be considered part of the set of running pods for replication controllers even pods represent running processes on nodes in the cluster. 

@xingxingxia @tnozicka  Please help review , thanks.

### CODE REVIEW

**Comment by zhouying7780** [comment]
Auto for test case: https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitems/testcase?query=subteam.KEY%3Amaster%20AND%20id%3AOCP%5C-19922. 

The checkpoint is : Terminating pods should not be considered part of the set of running pods for replication controllers even pods represent running processes on nodes in the cluster.

File: test/extended/deployments/deployments.go [hunk 1]
```diff
@@ -1549,4 +1550,64 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
...
 		})
 	})
+
+	g.Describe("running pods [Conformance]", func() {
+		dcName := "nettest"
+		g.AfterEach(func() {
+			failureTrap(oc, dcName, g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("Terminating pod should not be considered part of the set of running pods", func() {
```
Comment (tnozicka) [inline test/extended/deployments/deployments.go]: I don't think this is a test for DC. The accounting of pods is copied over from ReplicaitionController so the test would be suited to be placed there but it looks more like a unit test

**Pull request closed without merging**